docs/src/gs/create-workload-cluster.md

```diff
@@ -8,15 +8,19 @@ Choose one of the available templates for to create your workload clusters from
-| `OCI_SHAPE_OCPUS`              | 1                   | The number of OCPUs allocated to the instance                                                                                                                                                                                                                                                                                                                                                             |
-| `OCI_SSH_KEY`                   |                     | The public SSH key to be added to the Kubernetes nodes. It can be used to login to the node and troubleshoot failures.                                                                                                                                                                                                                                                                                    |
-| `OCI_PV_TRANSIT_ENCRYPTION`     | true                | [In-transit encryption](https://docs.oracle.com/en-us/iaas/Content/File/Tasks/intransitencryption.htm) provides a way to secure your data between instances and mounted file systems using TLS v.1.2 (Transport Layer Security) encryption. Only [some bare metal instances](https://docs.oracle.com/en-us/iaas/releasenotes/changes/60d602f5-abb3-4639-aa19-292a5744a808/) support In-transit encryption |
+| Parameter                                 | Default Value       | Description                                                                                                                                                                                                                                                                                                                                                                                                    |
+|-------------------------------------------|---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `OCI_COMPARTMENT_ID`                      |                     | The OCID of the compartment where the OCI resources are to be created                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
"Are to be created" is a little cumbersome. Would "in which the resources are created" work better perhaps?